node/src/actors/chain_manager/mod.rs

```diff
@@ -1466,7 +1446,7 @@ impl ChainManager {
                 next_state
             ),
         }
-
+        self.notify_change_node_status(next_state);
```

Comment:
This will be executed even if the state does not change, we can check that `old != new` to avoid sending redundant notifications, or we can leave it like this if it's not a problem.

Reply:
I guess that redundant notifications don't do any harm, and we could speculate that maybe they could be even helpful in some cases. I'm for leaving it in here and just wait and see if they are too spammy.